spec/move-tags-spec.js

```diff
@@ -0,0 +1,213 @@
+// var tidy = require('../mustache-tidy.js').mustacheTidy;
+var tidy = require('../');
+
+describe('Moving tags', function() {
```

Comment:
@jasny Yeh, I know. Seems like I did this in a hurry, copy/pasting first test and changing it. I'll fix this.